### PR TITLE
add an English readme

### LINKED ISSUE #139: Add English readme
**What would you like to be added**:
An English readme
**Why is this needed**:
We are targeting global customers in the future.

### PR DESCRIPTION
**Pull request by lvtaococo**
Signed-off-by: lvtaococo <lvtao_yewu@cmss.chinamobile.com>

#### What type of PR is this?

docs
#### What does this PR do?

Add English readme and modify pull request template
#### Which issue(s) does this PR fix?

Fixes #139 
#### Special notes for your reviewer:

Is the grammar, semantics, words, and spelling correct?
#### Screenshots (if applicable):

```release-note

```

#### Additional Comments:

```release-note
Kosmos version: v0.1.7
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,84 @@
...
-- `Multi-Cluster-Coredns`: 实现多集群服务发现；
-- `Elector`：负责gateway节点选举；
+- `Controller-Manager`：A component is used to gather network information from the current cluster, and it monitors changes in network settings.
+- `Network-manager`：A component is responsible for calculating the network configurations required for each node.
+- `Agent`：This is a DaemonSet used for configuring the host network, including tasks such as tunnel creation, routing, NAT, and so on.
+- `Multi-Cluster-Coredns`: This component facilitates multi-cluster service discovery.
+- `Elector`：It is responsible for conducting gateway node elections.
 
-### 快速开始
+### Quick Start
```
Comment (ONE7live) [inline README.md:24]: It is recommended that the quick start content be updated based on the official account article, what do you think?
Can be used as a separate chapter to describe quickly starting Kosmos, no longer limited to clusterlink.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,84 @@
 # KOSMOS
 
-Kosmos是移动云开源的分布式云原生联邦集群技术的集合，其名称kosmos：k代表**k**ubernetes，c**osmos**表示宇宙（希腊语），寓意kubernetes的无限扩展。目前，kosmos主要包括三大模块，分别是：**多集群网络**、**多集群管理编排**、**多集群调度**。此外，kosmos还配备一款kosmosctl工具， 可以快速进行kosmos组件部署、添加集群、测试网络连通性等工作。
+Kosmos is an open-source collection of distributed cloud-native federated cluster technologies. The name "kosmos" combines 'k' representing Kubernetes and 'cosmos' which means universe in Greek, symbolizing the limitless expansion of Kubernetes. Currently, Kosmos primarily consists of three major modules: ClusterLink, ClusterTree and Scheduler. Additionally, Kosmos is equipped with a tool called kosmosctl, which allows for quick deployment of Kosmos components, adding clusters, and testing network connectivity.
```
Comment (hanweisen) [inline README.md]: May be this is better, What do you think?
Kosmos is an open-source, all-in-one distributed cloud-native solution.

@wuyingjun-lucky @xyz2277  @duanmengkk @wangyizhi1

File: docs/images/kosmos-WechatIMG.jpg
Comment (zhaohuiweixiao) [inline docs/images/kosmos-WechatIMG.jpg:1]: Whether the group name needs to be in English？

File: README.md (hunk 2 above)
Comment (wuyingjun-lucky) [reply README.md]: It seems right

File: docs/images/kosmos-WechatIMG.jpg
Comment (wuyingjun-lucky) [reply docs/images/kosmos-WechatIMG.jpg:1]: It is ok

File: README.md (hunk 2 above)
Comment (lvtaococo) [reply README.md]: I have got it

**Pull request merged**